### DISCUSSION TITLE
Satellites not shown

### DISCUSSION THREAD

**Discussion by Wellco** [Stellarium/stellarium#2986] [General]
I am intersted to understand the impact of the Starlink constellation on my astrophotographery. I've loaded the satellite details from here : http://www.celestrak.org/NORAD/elements/starlink.txt (I don't know about this source, but I think it is correct)

![Screenshot 2023-01-10 at 17 52 07](https://user-images.githubusercontent.com/5941720/211625906-9f07b5f1-dff9-4d88-8a2b-96b9cd83e38b.png)

I tried toggling the button on and off at the bottom of the screen, but without success. I also tried combinations with/without the ground and the atmosphere. I've done Windows update (Windows 11) and restated Stellarium a few times. The satellites never show up

My PC says that it is running OpenGL 4.6 so I think that's OK

The only other thing that I can think of that might be a problem is I am running the PC headless from a Mac using MS Remote Desktop.  I hope this isn't the issue as there's no chance of running the PC with a physical  monitor.

Any suggestion will be greatly appreciated 

Tim

**Comment by alex-w** [comment]
Just for simple experiment:
a) Click on a "checkbox" button (left bottom corner) to select all satellites and mark "Displayed" checkbox in right side.
b) Switch to first tab - Settings
c) Mark "Show satellites as markers" option

What you see on the screen?

**Comment by Wellco** [comment]
Hi Alexander,

I missed the check box for ’Show Satellites as Marker’. Ticked as you suggest and hey presto.

Thanks for your help.

Tim
